Section3

tatsuya167 opened the pull request.

wakandajp commented:
<img width="1680" alt="Screen Shot 2019-10-11 at 9 58 35 AM" src="https://user-images.githubusercontent.com/15808541/66670161-ead3f480-ec0d-11e9-8bf2-29449778fd42.png">
I can't see anything in section3, can you tell me what is this pull request for?

tatsuya167 commented:
> <img alt="Screen Shot 2019-10-11 at 9 58 35 AM" width="1680" src="https://user-images.githubusercontent.com/15808541/66670161-ead3f480-ec0d-11e9-8bf2-29449778fd42.png">
> 
> I can't see anything in section3

>Because I wanted to solve conflict , I deleted some html tag which I added. I guess now you can see somthing.

wakandajp commented:
still in progress, please pull request again after done
